### PR TITLE
Update simple-db.md - format Error variants as code

### PR DESCRIPTION
**Pull request by amanjeev**
Add backticks

### CODE REVIEW

**Review by jonathanpallant — APPROVED** [review]

**Review by miguelraz — APPROVED** [review]

**Comment by jonathanpallant** [comment]
Wait, did you rebase on main and wait for a green light before merging? I need to tick the box that forces you to do that.

**Pull request merged**
